Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1014,6 +1014,27 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
     return (y, { v in Raw.eluGrad(gradients: v, outputs: y) })
 }
 
+/// Returns a tensor by applying the leaky ReLU activation function to the specified tensor element-wise.
+/// Specifically, computes `max(features, features * alpha)`.
```

Comment:
`features` is undefined.

```suggestion
/// Specifically, computes `max(x, x * alpha)`.
```